README.md

```diff
@@ -1 +1,108 @@
-# morse-code
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [🎯 Decode a Morse Code Message](#-decode-a-morse-code-message)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [👋 Show your support ](#-show-your-support-)
+  - [🔭Acknowledgments ](#acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 🎯 Decode a Morse Code Message<a name="about-project"></a>
+
+This project featues basic functionality on how to decode a morse-coded message
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Decode morse Char**
+- **Decode morse word**
+- **Decode morse message**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+you have to install those in your local machine.
+
+- [ ] Ruby.
+- [ ] A Text Editor.
+
+### Setup
+
+Clone the project.
+
+```bash
+  git clone "git@github.com:Katended/morse-code.git"
+```
+
+Go to the project directory.
+
+```bash
+  cd morse-code
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👤 Authors <a name="author"></a>
+
+👤 **Author1**
+👤 **Katende David**
+
+- GitHub: [@Katended](https://github.com/Katended/)
+- LinkedIn: [Katende David](https://www.linkedin.com/in/katendedavid/)
+
+👤 **Hussein Hashi**
+
+👤 **Author2**
+
+👤 **Hussein Hashi**
+
+- GitHub: [Husush23](https://github.com/husush23)
+- Twitter: [Hussein](https://twitter.com/HusseinKadare2)
+- LinkedIn: [Hussein](https://www.linkedin.com/in/husseinkadare/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Katended/morse-code/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👋 Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT) licensed.
```

Comment:
- [ ] As requested by the previous reviewer, The License link in the README filer is broken. Please make sure to fix this. It will make the README file more professional.
  - Replace the license link with this. `https://github.com/Katended/morse-code/blob/decode-morse/MIT.md`.